src/style/core.ts

```diff
@@ -88,6 +89,23 @@ const root = css`
         ${darkTheme}
       }
     }
+
+    /* Add box-shadow on the last frozen cell */
+    :nth-last-child(1 of .${cellFrozen}) {
```

Comment:
Added here as header does not use `row` class. I can instead add it to both `row` and `header-row` if you prefer

Reply:
Added a variable instead. I think it is better to use more specific styles